src/Model/ImageLoading.py

```diff
@@ -1,11 +1,4 @@
 """
-Skeleton for the interface between new boot window and existing patient window.
-Intention is to replace, recreate, or reuse most of the functionality served by the ProgressBar.Extended class.
-The functions in this file should be enough to generate the arguments required to create a MainWindow class.
-read_data_dict, file_names_dict, rois, raw_dvh, dvh_x_y, dict_raw_contour_data, dict_numpoints, dict_pixluts.
-The reason why some of these functions will be directly copied is because they currently exist as member functions
-of ProgressBar.Extended class and that class is to be deprecated/refactored with the new patient selection window.
-
```

Comment:
This section has been removed as it is no longer relevant to the current state of the file.